bin/git-up

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+git fetch origin
+git rebase origin/master
```

Comment:
I could be wrong, but `pull` is functionally equivalent to `fetch && merge`, so isn't this equivalent to `git pull --rebase origin master`?